index.html

```diff
@@ -229,38 +229,71 @@ <h3>
             <dt id="data-integrity-1" class="spec">Verifiable Credential Data Integrity 1.0</dt>
             <dd>
               <p>
-This specification defines how to express proofs of integrity, such as digital
-signatures or proofs of existence, for bounded documents, such as verifiable
-credentials. Concrete serializations will be provided based on <a
-href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>, the <a
-href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>, 
-the <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1
-Cryptosuite</a>, and for <a href="https://github.com/w3c-ccg/lds-jws2020/">
-JSON Web Signature 2020</a>. Concrete serializations based on the <a
-href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a> might be provided, if an <a
-href="https://identity.foundation/bbs-signature/">IETF BBS+ RFC</a> is published
-in time. Concrete serializations for VC-JWP might be provided, if an <a
-href="https://json-web-proofs.github.io/json-web-proofs/">IETF JWP RFC</a> is
-published in time. 
-The final output specifications might not map directly to
-the list of input drafts. Other output serializations covering cryptographic
-primitives published in IETF RFCs not mentioned in this paragraph might be
-produced.
+This family of specifications defines how to express proofs of integrity for
+verifiable credentials using a number of concrete serializations for each of the
+defined syntaxes. The specific set of concrete serializations included will be
+determined by the Working Group. The following are a non-exhaustive selection
+of expected input documents:
               </p>
-
-              <p class="draft-status"><b>Draft state:</b>
-                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>,
+              <p class="input-documents"><b>Container Formats:</b>
                 <a href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>,
-                <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a>,
-                <a href="https://github.com/w3c-ccg/lds-jws2020/">JSON Web Signature 2020</a>              
+                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>
```

Comment:
```suggestion
                <a href="https://json-web-proofs.github.io/json-web-proofs/">VC-JWP</a>, 
                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>
```

Reply:
@OR13, this PR introduces a new section that matches the way the WebApps WG structured their charter (which has been repeatedly approved by W3C AC):

https://www.w3.org/2019/05/webapps-charter.html#deliverables

Their charter lists two classes of deliverables:

1. Normative deliverables that they definitely plan to deliver.
2. Deliverables that they intend to become normative IF the incubation group / external dependency is done on a reasonable timeframe.

For our charter, the VC-JWP stuff falls into the second category, and is why it's listed down there instead of here. If we put it in this list, we're saying we intend to take something normative that has no finalized spec at IETF to build upon.

For that reason, I can't take this change request. Can you either 1) retract it, or 2) provide justification for putting VC-JWP in "we're definitely going to do this" normative scope when the IETF work hasn't been completed?

Reply:
Ahh I see, in that case, I retract the change request... but still have concerns about the crypto suites as the relate to 1, or 2.

but I think the restructuring is so desperately needs I will hold on any further comments, until we have it.